websites/K/Kingdom of Loathing/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+        "name": "gausie",
+        "id": "263438077865754644"
+    },
+    "service": "Kingdom of Loathing",
+    "altnames": ["KoL", "KoLmafia"],
+    "description": {
+        "en": "Presence for the Kingdom of Loathing"
+    },
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/FBkmCGO.gif",
+    "thumbnail": "https://i.imgur.com/FBkmCGO.gif",
```

Comment:
logo must be 512x512
thumbnail most be a promotion banner or a screenshot of the website homepage